### PR TITLE
Round kernel

### PR DESCRIPTION
**Pull request by andrewpareles**
round.out hammerblade kernel & tests

### CODE REVIEW

File: aten/src/ATen/native/hammerblade/Round.cpp [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+#include <ATen/native/TensorIterator.h>
+#include <ATen/native/UnaryOps.h>
+#include <ATen/native/hammerblade/Offload.h>
+
+namespace at { namespace native {
+namespace {
+
+
+static void round_kernel_hb(TensorIterator& iter) {  
+  // TODO: replace with
```
Comment (yodada) [inline aten/src/ATen/native/hammerblade/Round.cpp]: We have some comments leftover here ;)

File: hammerblade/torch/tests/test_round.py [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+    assert got_device == torch.device("hammerblade")
+    assert torch.equal(got_tensor, expected_tensor)
+
+def test_torch_round_1():
+    x = torch.Tensor([-2.5, -2, -1.5, -1, -.5, 0, .5, 1, 1.5, 2, 2.5, 3.5, 4.5, 5.5, 6.5]) 
+    _test_torch_round(x)
+
+def test_torch_round_2():
+    x = torch.randn(2, 3)
+    _test_torch_round(x)
```
Comment (yodada) [inline hammerblade/torch/tests/test_round.py]: I guess flake8 is complaining about this no newline thing?

File: hammerblade/torch/tests/test_round.py [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+
+def _test_torch_round(x): #x is a torch.Tensor
+    expected_tensor = x.round()
+    
+    h = x.hammerblade()
+    got_hb = h.round()
+    got_device = got_hb.device
+    got_tensor = got_hb.cpu()
+
+    print("tensor to round:\n", x)
```
Comment (yodada) [inline hammerblade/torch/tests/test_round.py]: It would be nice to remove the prints ;)

**Review by yodada — APPROVED** [review]
Sweet! If you could resolve the "no newline" issue, I think it looks all good to me!

**Comment by andrewpareles** [comment]
Alright-- I think this should fix it!

**Comment by yodada** [comment]
Sweet! Thanks for the kernel! Merging now ...

**Pull request merged**